.github/workflows/dockerbuild.yml

```diff
@@ -146,8 +150,8 @@ jobs:
            SLACK_CHANNEL: getintoteaching_tech
            SLACK_COLOR: '#3278BD'
            SLACK_ICON: https://github.com/rtCamp.png?size=48
-           SLACK_MESSAGE: ':disappointed_relieved: Pipeline Failure carrying out job ${{github.job}} :disappointed_relieved:'
-           SLACK_TITLE: 'Failure: ${{ github.workflow }}'
+           SLACK_MESSAGE: 'The pipeline has failed to deploy the API to the development environment'
```

Comment:
Is it worth still including the `github.job` in the message to help tracking back to where failures are?

Reply:
I was making the wording more descriptive, the job_id just gives deploy_dev and i thought that wasnt that helpful, when you navigate to the workflow it is pretty obvious